tests/conftest.py

```diff
@@ -93,6 +93,10 @@ def pytest_addoption(parser):
                      help="Allow recovery attempt in sanity check in case of failure")
     parser.addoption("--check_items", action="store", default=False,
                      help="Change (add|remove) check items in the check list")
+    parser.addoption("--post_check", action="store_true", default=False,
+                     help="Perform post test sanity check if sanity check is enabled")
+    parser.addoption("--post_check_items", action="store", default=False,
```

Comment:
So now with your change, the `post_check` is default off, right?

Reply:
Both before and with this change, `post_check` is default off. The default behavior is not changed.